hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSink.java

```diff
@@ -274,8 +276,10 @@ public void replicateEntries(List<WALEntry> entries, final CellScanner cells,
       this.metrics.setAgeOfLastAppliedOp(entries.get(size - 1).getKey().getWriteTime());
       this.metrics.applyBatch(size + hfilesReplicated, hfilesReplicated);
       this.totalReplicatedEdits.addAndGet(totalReplicated);
+      // TODO(baugenreich) -> does it make sense to reset the counter here ??
```

Comment:
If we are using a counter and not a gauge does it make sense to reset the counter to zero when the replication edits are applied correctly?

Reply:
no, the counter is a lifetime thing. the way counters usually are used is either by turning them into a rate or a delta. so if batches are succeeding, the rate/delta will just show up as zero.